src/test/java/edu/hawaii/its/groupings/service/EmailServiceTest.java

```diff
@@ -49,6 +49,7 @@ public void send(SimpleMailMessage mailMessage) throws MailException {
 
         emailService = new EmailService(sender);
         emailService.setEnabled(true);
+        emailService.setTo("address");
```

Comment:
Is creating the setter `setTo` needed for this unit test? Could you check the JaCoCo test coverage for this test with and without the setter? Here's the documentation: https://uhawaii.atlassian.net/wiki/spaces/SITARd/pages/12943618/jaCoCo+Code+Coverage

Reply:
Without the setter, I would have to add a null check in the send methods of EmailService when it checks if the recipient email != the team's email list. Below are screenshots of the Jacoco reports.
Main: 
![image](https://user-images.githubusercontent.com/77418067/233510939-6c342a8f-c583-46fd-8f58-43f6422268d5.png)

No setTo
<img width="728" alt="image" src="https://user-images.githubusercontent.com/77418067/233510988-4324df16-a5f9-4923-bd89-328670ce0903.png">

setTo
<img width="715" alt="image" src="https://user-images.githubusercontent.com/77418067/233511034-6ba7c84b-96ea-464f-8eed-37da08c8b86b.png">

Reply:
Could you try to get the test coverage to 100% or as high as you can? Thanks.

Reply:
![image](https://user-images.githubusercontent.com/77418067/233748766-151690af-2b56-4f8a-bcde-4be0ea2a7c7d.png)
I couldn't figure out how to force a unit test down the path where the InetAddress.getLocalHost() throws an UnknownHostException

Reply:
Looks great. Also, could you switch the Hamcrest asserts into Junit assertions? So instead of using assertThat from Hamcrest, we use assertEquals. Note that in assertEquals, the first parameter is the expected value.